fakenet/listeners/ProxyListener.py

```diff
@@ -0,0 +1,397 @@
+import socket
+import SocketServer
+import threading
+import sys
+import glob
+import time
+import importlib
+import Queue
+import select
+import logging
+import ssl
+from OpenSSL import SSL
+from ssl_utils import ssl_detector 
```

Comment:
Update setup.py to include the new ssl_utils library:

    package_data={'fakenet': ['*.pem','diverters/*.py', 'listeners/*.py', 'listeners/ssl_utils/*', 'configs/*.ini', 'defaultFiles/*', 'lib/64/*', 'lib/32/*']}

otherwise ssl_utils is not available in this module